nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -288,6 +288,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 - `services.dhcpcd` service now don't solicit or accept IPv6 Router Advertisements on interfaces that use static IPv6 addresses.
   If network uses both IPv6 Unique local addresses (ULA) and global IPv6 address auto-configuration with SLAAC, must add the parameter `networking.dhcpcd.IPv6rs = true;`.
 
+- The module [services.ankisyncd](#opt-services.ankisyncd.package) has been switched to [anki-sync-server-rs](https://github.com/ankicommunity/anki-sync-server-rs) from the old python version, which was difficult to update and not updated in a while, not supporting recent clients. The config file is different, but users' database is compatible so at most a rsync from clients might be needed.
```

Comment:
Couple wording nits:

1. "rsync" -> "resync"
2. "which was difficult to update and not updated in a while, not supporting recent clients" -> "which was difficult to update, had not been updated in a while, and did not support recent versions of anki." (the last bit is a sentence fragment otherwise I think).